### PR TITLE
Update texts and layout

### PR DESCRIPTION
**Pull request by Kebap**
From 
```
You can find the test versions here:
- (coming soon) (linux)
```

To:
```
You can directly test the changes here:
- LINUX: (the download link will be here in a few minutes)
```

I would have liked `Linux` instead, but `Osx` seems too ugly.

Alternatively after including the feedback:
```
You can directly test the changes here:
- linux: Download pending. Check back soon!
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ const updateCommentUrlFromLog = (os,log, comment) => {
   }
   const deployUrl = matches[1]
   application.log(`Deployed the output for ${os} to: ${deployUrl}`)
-  comment.body = comment.body.replace(new RegExp(`- .+? \\(${os}\\)`), `- ${deployUrl} (${os})`)
+  comment.body = comment.body.replace(new RegExp(`- ${os}: .+?`), `- ${os}: ${deployUrl}`)
```
Comment (keneanung) [inline index.js]: You'll need to change the case of the content of the variable `os` as well:
```suggestion
  comment.body = comment.body.replace(new RegExp(`- ${os}: .+?`, "i"), `- ${os.toUpperCase()}: ${deployUrl}`)
```

**Comment by keneanung** [comment]
To be honest, I'm not sure I like this change. Making the OS names uppercase seems unfriendly :disappointed:

**Comment by vadi2** [comment]
I think the current template is better, too. Not into the uppercasing and a few minutes might not be a few minutes depending on how many builds there are to do...

**Comment by Kebap** [comment]
OK removing the uppercase then.
How do you feel about moving the OS to the beginning of the line, instead of the long end in brackets? I feel it directs traffic better.
The problem I have with "coming soon" is it could be interpreted as "feature is not implemented yet", whereas it is meant for "link will be included after compilation has finished"

**Comment by vadi2** [comment]
`(building, check back soon)` ?

**Comment by Kebap** [comment]
I found the nice word `pending` which seems to summarize the situation quite nicely.

**Comment by Kebap** [comment]
> Modifying the open PRs would probably be the best thing

On it.

**Comment by Kebap** [comment]
I checked all 19 open PR and modified all 5 with a comment by your bot into the new format:

`- windows: random text`

File: index.js (hunk 1 above)
Comment (keneanung) [inline index.js]:
```suggestion
  comment.body = comment.body.replace(new RegExp(`- ${os}: .+`), `- ${os}: ${deployUrl}`)
```

**Pull request merged**